### ISSUE TITLE
Edit of only image alt text in GtS does not work

### ISSUE THREAD

**Issue by mirabilos** [feditext/feditext#480]
**Describe the bug**
An attempt to edit a post and change only the alt text of an image fails with Unprocessable Entity. I reported this against GtS first as https://github.com/superseriousbusiness/gotosocial/issues/3817 but this apparently works with multiple other clients, so the current idea is that FediText doesn’t send the change correctly if the post body is not changed.

**To Reproduce**
1. Go to a toot you wrote yourself that has an image
2. Click on … and Edit
3. Click on the image
4. Edit the image alt text
5. Submit the edit without changing anything in the post body
6. See error

**Expected behavior**
Editing posts should work no matter whether I changed only one of post body or image alt text or CW. (I think it should NOT work if the post has a poll because edits to posts with polls lose all existing votes.)

**(please complete the following information):**
 - Device: iPhone SE 2022
 - OS: iOS 18.3.1
 - App Version: 1.7.2-130

**Additional context**
The JSON error is pasted in the GtS bug.
